app/src/lib/emails/InviteUserTemplate.tsx

```diff
@@ -15,19 +17,13 @@ import {
 } from "@react-email/components";
 
 export default function InviteUserTemplate({
-  url,
-  user,
-  city,
-  invitingUser,
-  members,
+  url = "/",
+  user = { name: "" },
 }: {
   url?: string;
-  user?: { name: string; email: string; cityId?: string };
-  city?: CityAttributes;
-  invitingUser?: { name: string; email: string };
-  members: UserAttributes[];
+  user?: { name: string };
 }) {
-  const countryCode = city?.locode?.split(" ")!;
+  const generatedCode = "2KGuWexgH@!";
```

Comment:
We should probably generate this using base64 or another approach to generate random codes. And we should do that outside of the email template and save it.